lib/dealership.rb

```diff
@@ -0,0 +1,13 @@
+class Dealership
+  attr_reader :inventory
+  def initialize(dealership, address)
+    @dealership = dealership
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    0
+  end
```

Comment:
This is a start, but I would calculate the inventory_count from @inventory so:
```suggestion
  def inventory_count
    @inventory.length
  end
```